scripts/test.sh

```diff
@@ -31,8 +31,8 @@ GINKGO_ARGS="-nodes=$PARALLELISM" kubernetes-test.e2e \
   -host $SERVER \
   -ginkgo.focus $FOCUS \
   -ginkgo.skip "$SKIP" \
-  $EXTRA_ARGS \
-  -report-dir $ACTION_JUNIT 2>&1 | tee -a $ACTION_LOG
+  ${EXTRA_ARGS[@]} \
+  -report-dir $ACTION_JUNIT 2>&1 | tee -a $ACTION_LOG \
```

Comment:
i don't think you need this last back tick

```suggestion
  -report-dir $ACTION_JUNIT 2>&1 | tee -a $ACTION_LOG
```